modules/ballerina-core/src/test/java/org/ballerinalang/core/parser/negative/InvalidSyntaxParserTest.java

```diff
@@ -20,6 +20,7 @@
 
 import org.ballerinalang.core.utils.BTestUtils;
 import org.ballerinalang.util.exceptions.BallerinaException;
+import org.ballerinalang.util.exceptions.ParserException;
 //import org.ballerinalang.util.exceptions.ParserException;
```

Comment:
Please remove the commented out ParserException. I think that commenting out is useless. Needs to be scrapped.